src/Extensions/azure-storage/azure-storage/AzureBlobEgressProvider.AutoFlushStream.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Egress.AzureBlob
+{
+    partial class AzureBlobEgressProvider
```

Comment:
Please do not copy code wholesale. Either move it to the new location or link it in. If you need to add a temporary ProjectReference to dotnet-monitor, I would be fine with that too.